packages/travix-ui-kit/components/badge/badge.scss

```diff
@@ -240,6 +240,10 @@ $ui-badge__arrow-border-fix: calc(var(--tx-badge-border-bottom-width) * var(--tx
         border: none;
```

Comment:
What do you think to migrate this file to CSS since you are working on Badge component?

Reply:
we can plan it in coming sprints